asm/bss_800029B0.s

```diff
@@ -229,7 +229,7 @@ glabel D_8015F8E4
 glabel D_8015F8E8
 .skip 8
 
-glabel D_8015F8F0
+glabel gPlayerPositionLookup
```

Comment:
Is this a lookup table? If so just put LUT.

Reply:
Done